omero/sysadmins/server-upgrade.txt

```diff
@@ -368,6 +368,18 @@ fixes which require it.**
 If necessary ensure you have set up a regular task to clear out any stale
 OMERO.web session files as described in :ref:`omero_web_maintenance`.
 
+OMERO.web migrating from Apache to Nginx
+""""""""""""""""""""""""""""""""""""""""
+
+Since OMERO 5.2.6 support for Apache and mod_wsgi deployment is deprecated.
+It is recommended using :doc:`/sysadmins/unix/install-web/install-nginx` or
```

Comment:
"It is recommending using ..." reads weirdly to me. "It is recommended to use ..." or "We recommend using ..." perhaps?